assets/src/edit-story/components/checklist/constants.js

```diff
@@ -457,14 +456,7 @@ export const PRIORITY_COPY = {
     ),
   },
   videoNotOptimized: {
-    title: __('Optimize videos', 'web-stories'),
-    footer: sprintf(
-      /* translators: %s: video resolution (720p) */
-      __(
-        'Videos larger than %s can cause slower loading and higher bandwidth costs.',
-        'web-stories'
-      ),
-      `${MAX_VIDEO_RESOLUTION}p`
-    ),
+    title: __('Videos not optimized', 'web-stories'),
+    footer: __('Unoptimized videos may cause playback issues.', 'web-stories'),
```

Comment:
where did this copy get sourced from? I remember Will making a last minute update to the video optimize copy for a previous release.

Reply:
This is from the latest figma. I'll assume that your PR here: https://github.com/google/web-stories-wp/pull/8034/files is correct and revert this change unless otherwise instructed.